caffe2/CMakeLists.txt

```diff
@@ -1049,44 +984,28 @@ if(USE_ROCM)
   # Call again since Caffe2_HIP_INCLUDE is extended with ATen include dirs.
   hip_include_directories(${Caffe2_HIP_INCLUDE})
 
-  filter_list(__caffe2_hip_srcs_cpp Caffe2_HIP_SRCS "\\.(cu|hip)$")
```

Comment:
@bddppq - not sure what it was for, please double check

Reply:
This (combined with next line that sets the src property) is for using hcc to compile only .cu and .hip files, and using host compiler for other gpu files that don't have gpu kernels (e.g. _gpu.cc).
This line seems to have been moved [above](https://github.com/pytorch/pytorch/pull/20774/files#diff-8f885316d57c59b5aa82e9857dbe5935R529) (instead of removed).